packages/fast-components-class-name-contracts-msft/src/button/index.ts

```diff
@@ -24,6 +24,11 @@ export interface ButtonClassNameContract extends BaseButtonClassNameContract {
      */
     button__justified?: string;
 
+    /**
+     * The stealth appearance modifier
```

Comment:
I wonder if we can add what "stealth" actually means here? Coming from the outside and not being a part of this design conversation, I'm not quite sure what stealth really means.

Reply:
@nalogiudice I guess one could argue the same for the other appearances. This aligns with our color system. Maybe we need to add something in the Read Me that describes when to use each kind of appearance.